tasks/computer-vision/super-resolution/README.md

```diff
@@ -0,0 +1,55 @@
+# Super Resolution using convolutional neural networks.
+
+Increase image resolution x2 or x4 while maintaining or even improving image quality.
+
+### Important Note:
+
+Make sure to first download the models from the provided links:
+
+Super Resolution x2: https://drive.google.com/drive/folders/1Su9LjUUOarwRasRsnJvPBnJN34g0aawr?usp=sharing Super Resolution x4: https://drive.google.com/drive/folders/17jlTPybEkomlTc7kh75Jiy5fv886Uqcd?usp=sharing
+
+Download the whole folders 'pretrained_super_res_f4' and 'pretrained_super_res_f2' and paste them in 'mlflow_pretrained_models' without changing their names.
+
+## Usage
+
+### Train:
+
+List of parameters: 
+
+  data_path: Directory of training images. Make sure to keep all images in a folder named 'train' before running script. e.g. if images are in '/training_data/train/', data_path = '/training_data/'  
```

Comment:
When I tried to replicate result I got issue with path '/training_data/train/' however it worked with './training_data/train/'. Please make minor update on README.md.

Reply:
@lucky-singh, really appreciate your feedback.
I believe it's because you have to provide the full path of the 'training_data' folder.